### DISCUSSION TITLE
Refraction

### DISCUSSION THREAD

**Discussion by alessiav8** [xelatihy/yocto-gl#1307] [Q&A]
Salve professore, volevo chiederle se come risultato della refraction andasse bene
![refra](https://user-images.githubusercontent.com/77933342/139128115-4f944045-af80-4dbf-a990-e11022584839.jpg)
e

**Comment by xelatihy** [comment]
Non è corretto perchè non si vedono le immagini delle luci riflesse sul vetro.

**Comment by alessiav8** [comment]
Scusi professore ma cel shading posso implementarlo anche in raytrace
mettendo un tipo toon o necessariamente come nuovo shader?

Il ven 29 ott 2021, 09:15 Fabio Pellacini ***@***.***> ha
scritto:

**Reply by xelatihy** [reply]
si
